examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -114,44 +126,9 @@
 using namespace std;
 using namespace mfem;
 
-class ReducedSystemOperator;
-
-class HyperelasticOperator : public TimeDependentOperator
-{
-
-protected:
-    ParBilinearForm *M, *S;
-
-    CGSolver M_solver;    // Krylov solver for inverting the mass matrix M
-    HypreSmoother M_prec; // Preconditioner for the mass matrix M
-
-public:
-    HyperelasticOperator(ParFiniteElementSpace& f, Array<int>& ess_tdof_list_,
-                         double visc, double mu, double K);
-
-    /// Compute the right-hand side of the ODE system.
-    virtual void Mult(const Vector& vx, Vector& dvx_dt) const;
-
-    double ElasticEnergy(const ParGridFunction& x) const;
-    double KineticEnergy(const ParGridFunction& v) const;
-    void GetElasticEnergyDensity(const ParGridFunction& x,
-                                 ParGridFunction& w) const;
+#include "nonlinear_elasticity_global_rom_eqp.hpp"
 
-    mutable Vector H_sp;
-    mutable Vector dvxdt_sp;
-
-    ParFiniteElementSpace &fespace;
-    double viscosity;
-    Array<int> ess_tdof_list;
-    ParNonlinearForm* H;
-    HyperelasticModel* model;
-    mutable Vector z; // auxiliary vector
-    mutable Vector z2; // auxiliary vector
-    HypreParMatrix* Mmat; // Mass matrix from ParallelAssemble()
-    HypreParMatrix Smat;
-
-    virtual ~HyperelasticOperator();
-};
+class ReducedSystemOperator;
```

Comment:
Unused?

Reply:
Yes, used for implicit steps in the original MFEM code